CMakeLists.txt

```diff
@@ -0,0 +1,84 @@
+cmake_minimum_required(VERSION 2.8.12)
+
+cmake_policy(SET CMP0025 NEW)
+cmake_policy(SET CMP0028 NEW)
+cmake_policy(SET CMP0042 NEW)
+cmake_policy(SET CMP0048 NEW)
+cmake_policy(SET CMP0057 NEW)
+cmake_policy(SET CMP0076 NEW)
+
+list(APPEND CMAKE_MODULE_PATH
+     "${CMAKE_CURRENT_LIST_DIR}/cmake"
+     "${CMAKE_CURRENT_LIST_DIR}/cmake/packages"
+     "${CMAKE_CURRENT_LIST_DIR}/cmake/modules/share/modules/cmake")
+
+include(CMConfig)
+include(CMSetupVersion)
+
+if(NOT CMAKE_WORKSPACE_NAME OR NOT ("${CMAKE_WORKSPACE_NAME}" STREQUAL "actor"))
+    cm_workspace(actor)
+endif()
+
+macro(cm_find_package NAME)
+    foreach(ITERATOR ${CMAKE_WORKSPACE_LIST})
+        if(NOT "${NAME}" MATCHES "^${ITERATOR}_.*$" AND NOT "${NAME}" STREQUAL CM)
+            find_package(${ARGV})
+        else()
+            set(${ARGV0}_FOUND ON CACHE BOOL "")
+        endif()
+    endforeach()
+endmacro()
```

Comment:
Could you explain what is happening here and why we need this macro?

Reply:
It's extension for our cm_* cmake buildings. It can set that package found if we already find it for our project workspace. 

But I also have a question why it's macro and can we migrate it into this repo https://github.com/BoostCMake/cmake_modules/tree/master

Reply:
I decided to create a separate task for this, since I copied this CMake file from another project, and we need to make these changes everywhere: 

https://www.notion.so/nilfoundation/9a899d0028114e4f8ad8a54cf82faac5?v=d5f6f106a49a4384ab39be9346f5d73b&p=0aca2d5e953648ada1a96d60c345b3b4&pm=s

Please contribute to the description of this task if you want more changes, we will do it later, and in all repos at once.